content/docs/user-guide/experiment-management/index.md

```diff
@@ -46,7 +46,14 @@ To save an experiment, you can follow one of these roads:
 Experiments are saved locally by default but you can [share] them so that anyone
 can reproduce your work.
 
-## Metrics, plots, and parameters
+## Datasets and models
```

Comment:
Minor suggestion/not blocking: move it below metrics, plots, and parameters. I think those are still the first thing people want to know about when they start experiment tracking.

Reply:
Another minor/non-blocking suggestion: Datasets and models -> Models and datasets.